department-of-veterans-affairs/caseflow
# Remove Case Timeline header

Resolves #15578

### Description
Remove Case Timeline header

### Acceptance Criteria
- [ ] Header no longer exists under Case Timeline section

### Testing Plan
1. Go to the case details page of an appeal
2. Verify the header no longer appears under **Case Timeline** section 

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="845" alt="Screen Shot 2020-12-24 at 1 52 40 PM" src="https://user-images.githubusercontent.com/68879427/103107087-f41d6700-45ef-11eb-8b70-5041488b76bc.png">|<img width="846" alt="Screen Shot 2020-12-24 at 1 56 29 PM" src="https://user-images.githubusercontent.com/68879427/103107092-f97ab180-45ef-11eb-80ef-240a4a1b6769.png">
